run-ib-igprof

```diff
@@ -4,11 +4,20 @@ case $CMS_BOT_DIR in /*) ;; *) CMS_BOT_DIR=$(pwd)/${CMS_BOT_DIR} ;; esac
 WORKFLOWS=$1
 PROFILES=$2
 EVENTS=$3
+HPUWFS="23434.21"
 if [ "X$EVENTS" = "X" ] ; then EVENTS=1000; fi
 for prof in ${PROFILES} ; do
   mkdir -p $WORKSPACE/igprof/${prof}
   cd $WORKSPACE/igprof/${prof}
   runTheMatrix.py $WORKFLOWS --ibeos --command "-n $EVENTS --profile $prof" 2>&1 | tee runTheMatrix.log
+  for hpwf in $HPUWFS; do
+    if [[ $WORKFLOWS == *"${hpwf}" ]]; then
```

Comment:
@gartung , you do not need this check. The check on line 16 is enough to see if a HPWF was run or not

Reply:
updated